scvi/model/base/_rnamixin.py

```diff
@@ -84,13 +201,19 @@ def get_normalized_expression(
         -------
         If `n_samples` > 1 and `return_mean` is False, then the shape is `(samples, cells, genes)`.
         Otherwise, shape is `(cells, genes)`. In this case, return type is :class:`~pandas.DataFrame` unless `return_numpy` is True.
+        This function has two modes of operation:
+        If `n_samples` is provided or if `return_mean` is True,
+        this method returns a 3d tensor of shape (n_samples, n_cells, n_genes).
+        Otherwise, the method expects `n_samples_overall` to be provided and returns a 2d tensor
+        of shape (n_samples_overall, n_genes).
```

Comment:
can you clean this up with the text just above it?

Reply:
Done!